src/App.js

```diff
@@ -1,23 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Nav from "./components/Nav";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import ArticleList from "./components/ArticleList";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Home />
+      <Header />
+      <Nav />
+      <Routes>
+        <Route exact path="/" element={<Home />}></Route>
```

Comment:
The nice thing about `<Route>` components is that they can self close, like so...
```suggestion
        <Route exact path="/" element={<Home />} />
```